src/txmempool.h

```diff
@@ -94,7 +89,17 @@ class CTxMemPoolEntry
     uint64_t nSizeWithAncestors;
     CAmount nModFeesWithAncestors;
     int64_t nSigOpCostWithAncestors;
+    size_t nTxWeight;          //!< ... and avoid recomputing tx weight (also used for GetTxSize())
+    size_t nModSize;           //!< ... and modified size for priority
```

Comment:
In commit "Re-pack the CTxMempoolEntry to be closer to optimally packed":

This isn't used, guessing bad rebase.